editor/animation_track_editor.cpp

```diff
@@ -7653,6 +7696,14 @@ AnimationTrackEditor::AnimationTrackEditor() {
 
 	bottom_hf->add_child(selected_filter);
 
+	alphabetic_sorting = memnew(Button);
+	alphabetic_sorting->set_flat(true);
+	alphabetic_sorting->connect(SceneStringName(pressed), callable_mp(this, &AnimationTrackEditor::_sort_alphabetically_toggle));
```

Comment:
I wonder if it makes sense to add a new function (`_sort_alphabetically_toggle()`) to call already existing function  (`_update_tracks();`). Maybe connect it directly to `_update_tracks`?

Reply:
Yes good point. When I set this up I think I imagined more logic would go into that function. Removed it and connected directly to `_update_tracks()`